tutorials/audio/audio_buses.rst

```diff
@@ -6,31 +6,20 @@ Audio buses
 Introduction
 ------------
 
-Beginning with Godot 3.0, the audio engine has been rewritten from scratch.
-The aim now is to present an interface much friendlier to sound design
-professionals. To achieve this, the audio engine contains a virtual rack
-where unlimited audio buses can be created and, on each of it, unlimited
-amount of effect processors can be added (or more like, as long as your
-CPU holds up!)
+Godot's audio engine allows any number of audio buses to be created and any number of effect processors can be added to each bus. The processsor power of the device running your game will limit the number of buses and effects that can be used before performance starts to suffer.
```

Comment:
>The processsor power of the device running your game will limit the number of buses and effects that can be used before performance starts to suffer.

Reads somewhat like a non-sequitur to me after the sentence before. I understand why looking at the original of course, but still :) How about:

>***Only** the hardware* of the device running your game will limit the number of buses and effects that can be used before performance starts to suffer.

It's a small change, but IMO makes it flow better from the previous sentence.